bundles/org.openhab.binding.solarwatt/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -34,7 +34,7 @@
 
 		<label>Battery Converter</label>
 		<description>Battery converter to supply AC from battery storage.</description>
-
+		<semantic-equipment-tag>Battery</semantic-equipment-tag>
```

Comment:
Maybe not related to this PR, but why is a `Battery` not a child of a `PowerSupply` ? In this context it seems a `PowerSupply`

Reply:
Good question. I think there are the little (button) batteries in small sensor vs big solar inverter batteries. But we could probably reclassify the parentage without breaking anything..

Reply:
Done